homework21/tests/test_library.py

```diff
@@ -0,0 +1,168 @@
+"""Тестирование библиотеки (pytest)"""
+
+import os
+from loguru import logger
+from ..source.Homework11_1 import Reader, Book
+
+
+log_dir = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                       "library_log_folder")
+os.makedirs(log_dir, exist_ok=True)
+log_file = os.path.join(log_dir, "library_test.log")
+logger.remove()
+logger.add(log_file, level="DEBUG", enqueue=True)
```

Comment:
It is better this time, but I saw the same lines in `homework21/tests/test_bank_deposit.py` file. Better to use 1 file where you setup a logger, in test modules you will ned hust import it and use
https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application